body.tex

```diff
@@ -1,215 +1,336 @@
-The Github repository is tagged with the new issue number, \textit{v2.0}.
 
 
 \subsection{Test Documents}
 
-Based on the process just outlined above, the following documents are identified.
-
-These documents are generated using the \textbf{Docsteady}.
+The documents identified in the above steps are generated using the \textbf{Docsteady}.
 The generation can be done manually, or automatically.
-Authomation is particularly useful in case we want to see every day the progress of the previous day published in the lsst.io landing page.
-
-The extraction tool, Docsteady, permits to the user to concentrate only on the test activities,
-forgetting any documentation aspect.
+Automation is particularly useful in case we want to see daily progress published in the LSST the Docs landing page.
 
+The extraction tool ensures homogeneity of information at all levels in the Verification and Validation activities,
+and permits the user to concentrate only on the relevant test activities, without the need to worry about any documentation aspect.
 
-\subsubsection{Test Specification}
+ This is a summary of the test documents:
 
-The Test Specification is a document that baseline the test cases defined on a specific component.
-
-
-\subsubsection{Test Plan and Report}
-
-The Test Plan and Report include all planning and execution information. 
-The document  is issued in two times. The first issue corresponds to the consolidation of the planning activity, 
-the second to the finalized test campaign.
-
-
-\subsubsection{Verification Elements Baseline}
-
-This document provides a snapshot of the verification elements, which content is maintained in Jira issues, therefore it is very easy to change.
-Having a snapshot of the verification elements in a document, make it possible to assess them, approve and keep history.
+\begin{itemize}
+\item \textbf{Test Specification}:  baselines the test cases.
+A new issue of the document is made each time there is a substantial change in the test cases definition.
+\item \textbf{Test Plan and Report}:  for a single test campaign this includes all planning and execution information.
+It  is issued at least two  times. The first issue corresponds to the consolidation of the planning activity,
+the second to the finalized test campaign containing the results of the same.
+\item \textbf{Verification Baseline Document}: provides a snapshot of the verification elements.
+The verification elements are  maintained as Jira issues, and  thus are easy to change.
+Having a snapshot of the verification elements at a specific point in time  (in the form of a document) makes
+for a stable baseline and is good for tracking changes.
 Approved versions are uploaded in Docushare and used as a reference in other test documents.
+\end{itemize}
 
+Test Specifications and Verification Baseline documents are create for specific components, where components are the main products in a subsystem.
```

Comment:
Test Specifications and Verification Baseline documents are created for specific components, ...